apps/ledger-live-desktop/tests/specs/speculos/manage.ledgersync.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { test } from "../../fixtures/common";
+import { addTmsLink } from "tests/utils/allureUtils";
+import { getDescription } from "../../utils/customJsonReporter";
+import { AppInfos } from "tests/enum/AppInfos";
+
+// Use a single value instead of an array
+const app: AppInfos = AppInfos.LS;
+
+test.describe(`[${app.name}] Sync Accounts`, () => {
+  test.use({
+    userdata: "ledgerSync",
+    speculosApp: app,
+  });
+
+  test("Synchronize one instance then delete the backup", async ({ app, page }) => {
+    const description = getDescription(test.info().annotations).split(", ");
+    await addTmsLink(description);
```

Comment:
Maybe we should rework this part to have something like that:

```suggestion
test.describe("LedgerSync - Sync Accounts", () => {
  const app: AppInfos = AppInfos.LS;     // (Not shure if this line will work)

  test.use({
    userdata: "ledgerSync",
    speculosApp: app,
  });

  test(
    `Synchronize one instance then delete the backup`,
    {
      annotation: {
        type: "TMS",
        description: "B2CQA-000",
      },
    },
    async ({ app }) => {
```